src/interfaces.ts

```diff
@@ -18,7 +18,7 @@ export interface BeamButtonProps {
    */
   disabled?: boolean | ReactNode;
   /** If function, then it is the handler that is called when the press is released over the target. Otherwise if string, it is the URL path for the link */
-  onClick?: ((e: PressEvent) => void) | string;
+  onClick: ((e: PressEvent) => void) | string;
```

Comment:
I made this required, which ended up causing a bunch of updates needed to supply the `onClick` in many stories/tests that didn't include it yet.

Reply:
+1 to the change